modules/gatsby/src/components/styled/index.js

```diff
@@ -6,9 +6,16 @@ import ChevronDown from 'baseui/icon/chevron-down'
 import {styled} from 'baseui';
 /* eslint-disable import/prefer-default-export */
 
+import THEME from './theme';
```

Comment:
we should refrain from using './theme'. it is a remnant from the time when i had trouble getting baseweb working server side. the problem with what is in ./theme is that it's static and won't be overridden by parameters that the user may pass. meanwhile, the actual baseweb theme can be accessed from any component via the provider pattern.